README.md

```diff
@@ -166,6 +167,34 @@ docsSearchBar({
 });
 ```
 
+##### `Templating` <!-- omit in toc -->
+
+You can override the default mustache templates used by DocsSearch as long as they are formatted with mustache. To see all available templates that can be overridden go to [src/lib/templates.js](src/lib/templates.js).
+
+```javascript
+const mySuggestionTemplate = `
+    <a class="${suggestionPrefix}
+      {{#isCategoryHeader}}${suggestionPrefix}__main{{/isCategoryHeader}}
+      {{#isSubCategoryHeader}}${suggestionPrefix}__secondary{{/isSubCategoryHeader}}
+      "
+      aria-label="Hello World!"
+      href="{{{url}}}"
+      >
+      <div class="${suggestionPrefix}--category-header">
+          <span class="${suggestionPrefix}--category-header-lvl0">{{{category}}}</span>
+      </div>
+    </a>
+    `,
```

Comment:
This example is not really "true" because `suggestionPrefix` is not accessible if I'm not wrong.